sway/commands/resize.c

```diff
@@ -111,61 +104,48 @@ static void calculate_constraints(int *min_width, int *max_width,
 	}
 }
 
-static enum resize_axis parse_resize_axis(const char *axis) {
+static enum wlr_edges parse_resize_axis(const char *axis) {
 	if (strcasecmp(axis, "width") == 0 || strcasecmp(axis, "horizontal") == 0) {
-		return RESIZE_AXIS_HORIZONTAL;
+		return AXIS_HORIZONTAL;
```

Comment:
This value is outside the enum. The return type should be changed to `uint32_t`.

Reply:
Does that mean I should change everything from `enum wlr_edges` to `uint32_t` where it could contain `AXIS_HORIZONTAL` or `AXIS_VERTICAL`?

Reply:
Using `enum thing` with values outside of the enum is dangerous because `switch` statements might forget some cases. Bitfields are a special case of this. So yeah, everything should be changed if necessary.

Reply:
Done.